tests/test_server.py

```diff
@@ -111,6 +119,39 @@ def test_update_recommendation_not_found(self):
         resp = self.app.put('/recommendations/0', data=data, content_type='application/json')
         self.assertEquals(resp.status_code, status.HTTP_404_NOT_FOUND)
 
+    def test_dislike_recommendation(self):
+        """ Dislike a Recommendation """
+        new_recommendation = {}
+        data = json.dumps(new_recommendation)
+        resp = self.app.put('/recommendations/2/dislike', data=data, content_type='application/json')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        resp = self.app.get('/recommendations/2', content_type='application/json')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        new_json = json.loads(resp.data)
+        self.assertEqual(new_json['recommendation']['dislikes'], 2)
+
+    def test_dislike_recommendation_with_deletion(self):
+        """ Delete a Recommendation that is disliked """
+         # save the current number of recommendations for later comparrison
+        recommendation_count = self.get_recommendation_count()
+
+        new_recommendation = {}
+        data = json.dumps(new_recommendation)
+        resp = self.app.put('/recommendations/3/dislike', data=data, content_type='application/json')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        resp = self.app.get('/recommendations/3', content_type='application/json')
```

Comment:
Put a comment here though just indicating that dislikes hits 5 so its deleted. May make it easier to follow.

Reply:
Done